### DISCUSSION TITLE
Can you provide IL debugging function? I found that dnspy-64 bit. exe can be debugged, but I'm afraid I don't know whether there is a problem with the decompiled code

### DISCUSSION THREAD

**Discussion by gallop1** [icsharpcode/ILSpy#2488] [Help]
（您能提供下IL调试的功能吗？我在dnSpy-64位.exe发现可以调试，但是我怕不知道是不是反编译出来的代码是否有问题），因为这个是基于你写的软件设计出来的。Can you provide IL debugging function? I found that dnspy-64 bit. Exe can be debugged, but I'm afraid I don't know whether there is a problem with the decompiled code, because it is designed based on the software you wrote.另外我想知道我理解的正不正确，这个反编译出来的东西能简单理解成被机器代码人为按照你的爱好格式化一份类似于机器标准的代码吗？尽管很接近源代码了。但是我怕翻译出来的有问题，当然有误差还是可以接受的，就看有没有影响对每一个人的理解。In addition, I want to know whether I understand it correctly. Can this decompiled thing be simply understood as machine code formatting a code similar to machine standard according to your preference? Although very close to the source code. But I'm afraid there are problems with the translation. Of course, errors are acceptable, depending on whether they affect everyone's understanding.期望你能尽快回答这个问题。I hope you can answer this question as soon as possible. @siegfriedpammer，我觉得只有你能回答这个问题。I think only you can answer this question，我觉得只有你能回答这个问题，因为我感觉对比很多个软件，像Resharper、Refletor,dnSpy工具，只有你的比较接近，但是我个人也比较倾向于源代码，不过你的好像也不错。当然，很多用户也可以回答我这个问题，但我更是喜欢你能说下你的看法。I think only you can answer this question, because I feel that compared with many software tools, such as resharper, Refletor and dnspy, only yours is relatively close, but I personally prefer the source code, but yours seems to be good. Of course, many users can also answer my question, but I prefer you to say your views.

**Comment by christophwille** [comment]
No, a debugging feature is not planned at all.

**Reply by gallop1** [reply]
好的，如果没有，那么我有一个问题就是，我理解的反编译出来的代码是否可以理解成是机器可以自己识别的帮我们自动格式化的代码？OK, if not, I have a question: can the decompiled code I understand be understood as the code that the machine can recognize and help us format automatically?@Siegfried Pammer @siegfriedpammer
